src/MainWindow.vala

```diff
@@ -227,6 +229,17 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
             }
         });
 
+        var refresh_accellabel = new Granite.AccelLabel.from_action_name (
+            _("Refresh"),
```

Comment:
It might not be clear what this is refreshing, since someone might think it's reloading the current view? So slightly more explicit is probably better:

```suggestion
            _("Check for Updates"),
```